protos/bc.proto

```diff
@@ -24,24 +24,32 @@ message RpcTransaction {
 message RpcFeedTransaction {
     string owner_addr = 1;
     string feed_addr = 2;
```

Comment:
`owner_addr` and `feed_addr` ar OL addresses - which if these are `bytes` take up just half of the bytes (OL address in string representation = hex value)

Reply:
I could but am concerned a number of other protos like RpcTransaction store this as a string, perhaps to make command line arg submission easier. Otherwise much prefer anything smaller.

Reply:
Yeah, that is a bad decision made in past, let's not repeat it here. CLI can always send string like `0x3a2a7aaaaa7a7a2a3a4aaaaa678a48a5123a2a95` but and we make bytes (`Buffer` or `Uint8Array` in JS) our of it before creating a new `RpcFeedTransaction`.